src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.tsx

```diff
@@ -63,6 +62,12 @@ export const DefaultTranscriptListItem = (
   const transcriptStartX = scale(relativeTranscriptStart) as number;
   const transcriptWidth = scale(transcriptLength) as number;
 
+  const dispatch = useDispatch();
+
+  const handleToggle = () => {
```

Comment:
Something bothering me about the name `handleToggle`. Would `handleTranscriptClick` or `onTranscriptClick` be better?